templates/base.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Taxi Service</title>

```

Comment:
Consider adding a `{% block title %}Taxi Service{% endblock %}` around the title tag. This allows child templates to override the title block and set their own titles.